rskj-core/src/main/java/co/rsk/mine/MinerUtils.java

```diff
@@ -202,6 +203,9 @@
                 accountNonces.put(txSender, txNonce);
 
                 logger.debug("Accepted tx={} sender: {} value: {} nonce: {}", hash, txSender, txValue, txNonce);
+            } catch (NodeReferenceException nre) {
```

Comment:
we mostly use `e` or `ex` for naming exceptions - maybe I'd mantain that convention